### DISCUSSION TITLE
Articles

### DISCUSSION THREAD

**Discussion by trevorsaint** [ONSdigital/design-system#1066] [Feedback]
Use this place to discuss the [Articles](https://ons-design-system.netlify.com/components/articles/) in the [ONS Design System](https://ons-design-system.netlify.com)

**Comment by armstrongb** [comment]
Is there a variant of this without the images?

**Comment by trevorsaint** [comment]
@armstrongb It should work without an image, if you have not specified one. 

This is still in the works, so I can make sure it does.

**The patterns was designed and developed for Census2021 News**
https://deploy-preview-118--sdc-prototypes.netlify.app/prototypes/census-website-engage/news/
https://deploy-preview-118--sdc-prototypes.netlify.app/prototypes/census-website-engage/news/article/
https://deploy-preview-118--sdc-prototypes.netlify.app/prototypes/census-website-engage/news/category/

If you have any other requirements, let me know.

**Comment by armstrongb** [comment]
Not a requirement as such just a query. For wider use the ability to have the image optional would be useful I'd expect.

**Comment by armstrongb** [comment]
Also... is this the article or an article variant of card? Article to me would hint at the full content. What do you think @JNATION85 , @jrbarnes9 & @boxadesign ?
